docs/src/using-turing/guide.md

```diff
@@ -231,7 +279,7 @@ Inputs to the model that have a value `missing` are treated as parameters, aka r
 
 
 ```julia
-@model gdemo(x, ::Type{T} = Float64) where {T} = begin
+@model function gdemo(x, ::Type{T} = Float64) where {T}
```

Comment:
Can we add some doc on the new feature `sample(model, Prior(), N)`? I can't find it in the docs at the moment, but it might be I don't know where it is located.

Reply:
Good catch, I've added this.